app/main.py

```diff
@@ -1 +1,7 @@
-# write your code here
+def copy_file(text: str) -> None:
+    if text.split() == 3:
+        command, file_name_1, file_name_2 = text.split()
```

Comment:
Don't use `split` twice. You should create separate variable for storing result of `spilt` and use it where you need